app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +36,8 @@
             android:name=".circle_of_trust.Trustees"
             android:label="@string/title_activity_trustees"
             android:parentActivityName=".MainActivity"
-            android:windowSoftInputMode="stateHidden">
+            android:windowSoftInputMode="adjustPan"
```

Comment:
Seems like this PR is good to go. Can you update it with the fix I mentioned about?